accounts/views.py

```diff
@@ -78,5 +78,17 @@ def register_user(email, password, firstname, lastname, active ,role):
     return user
 
 
+def register_SIG(SIG,user,updated_at,status):
```

Comment:
How are you taking care that once a person has applied for the SIGs he's not able to apply again?
By the logic I feel a person will be able to apply again and again, how many times he wants

Reply:
Queried DB before entering